CI env for py2.6 cannot be prepared

```
Run sudo add-apt-repository ppa:deadsnakes/ppa
  
Cannot add PPA: 'ppa:~deadsnakes/ubuntu/ppa'.
ERROR: '~deadsnakes' user or team does not exist.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -40,7 +40,7 @@ jobs:
     - uses: actions/checkout@v3
     - name: apt-install python26
       run: |
-        sudo add-apt-repository ppa:deadsnakes/ppa
+        sudo add-apt-repository 'ppa:deadsnakes/ppa'
         sudo apt-get update
         sudo apt-get install python2.6
         sudo update-alternatives --install /usr/bin/python python /usr/bin/python2.6 1

